modules/ROOT/pages/_partials/include-alert-variables.adoc

```diff
@@ -0,0 +1,187 @@
+//
```

Comment:
TODO: NEED TO DETERMINE WHICH OF THE deployment types apply to Alerts that are going out.